addOns/dns/src/main/javahelp/org/zaproxy/addon/dns/resources/help/contents/dns.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<html>
+    <head>
+        <title>DNS</title>
+    </head>
+    <body>
+        <h1>DNS</h1>
+
+        <p>
+            Add-on to perform automatic DNS checks.
+        </p>
+
+        <h2>Current detections</h2>
```

Comment:
Better mention this is a scan rule (check other scan rule help to see what should include, e.g. link to code, alert ID).